consensus/state.go

```diff
@@ -1817,7 +1817,16 @@ func (cs *State) tryAddVote(vote *types.Vote, peerID p2p.ID) (bool, error) {
 					vote.Type)
 				return added, err
 			}
-			cs.evpool.AddEvidence(voteErr.DuplicateVoteEvidence)
+			var timestamp time.Time
+			if voteErr.VoteA.Height == 1 {
```

Comment:
This is going to break with non-zero genesis, so let's make sure we coordinate merging this and #5191.

Reply:
Shall I try merge this before or after #5191?

Reply:
Either is fine